packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorBlue600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  large: 12,
+  default: 6
+};
+
+const sizeToHeight = size => heights[size || 'default'];
+const sizeToBorderRadius = size => sizeToHeight(size) / 2;
+
+const ProgressBarContainer = styled.div.attrs(props => ({
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  margin: ${zdSpacingXs} 0;
+  border-radius: ${({ borderRadius }) => borderRadius}px;
+  background-color: ${zdColorGrey200};
+  color: ${({ color }) => color};
+`;
+
+const ProgressBarIndicator = styled.div.attrs(props => ({
+  height: sizeToHeight(props.size),
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  transition: width 0.1s ease-in-out;
+  border-radius: ${({ borderRadius }) => borderRadius}px;
+  background: currentColor;
+  width: ${({ value }) => value}%;
+  height: ${({ height }) => height}px;
+`;
+
+const Progress = ({ value, size, ...other }) => {
+  const percentage = Math.max(0, Math.min(100, value));
+
+  return (
+    <ProgressBarContainer
+      data-garden-id={COMPONENT_ID}
+      data-garden-version={PACKAGE_VERSION}
+      aria-valuemax="100"
+      aria-valuemin="0"
+      aria-valuenow={percentage}
```

Comment:
Should there be a default `aria-valuetext`? Either that, or throw an error if it's missing like https://github.com/zendeskgarden/react-containers/blob/5b06c16f04ee9792f002073e8e4e5a041a710f01/packages/accordion/src/useAccordion.js#L67

Reply:
We can just make it a required prop, I didn't know if you wanted to enforce that.

Reply:
Yeah, tbh I'm not exactly clear. From reading this – https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute – it may be that we should keep it as optional and just rely on `aria-valuenow` as the default.